doc/source/dev/contributor/meson_advanced.rst

```diff
@@ -62,6 +62,29 @@ twice, while ``build`` does support that.
         -I/path/to/include-dir
 
 
+Cross compilation
+=================
+
+Cross compilation is a complex topic, we only add some hopefully helpful hints
+here (for now). Please see
+`Meson's documentation on cross compilation <https://mesonbuild.com/Cross-compilation.html>`__
+for context.
+
+One common hiccup is that ``numpy``, ``pybind11`` and ``pythran`` require
+running Python code in order to obtain their include directories. This tends to
+not work well, either accidentally picking up the packages from the build
+(native) Python rather than the host (cross) Python or requiring ``crossenv``
+or QEMU to run the host Python. To avoid this problem, specify the paths to the
+relevant directories in your *cross file*:
+
+.. code:: ini
+
+    [properties]
+    numpy-include-dir = /abspath/to/host-pythons/site-packages/numpy/core/include
+    pybind11-include-dir = /abspath/to/host-pythons/site-packages/pybind11/include
+    pythran-include-dir = /abspath/to/host-pythons/site-packages/pythran
```

Comment:
The paths apparently need to be wrapped in single quotes, so the docs ought to reflect that in some way.
```suggestion
    numpy-include-dir = '/abspath/to/host-pythons/site-packages/numpy/core/include'
    pybind11-include-dir = '/abspath/to/host-pythons/site-packages/pybind11/include'
    pythran-include-dir = '/abspath/to/host-pythons/site-packages/pythran'
    
Remember to wrap these paths in single quotes.
```

Reply:
Right, this is a string value in the machine file and those do need quotes.

Reply:
Thanks, I included this and also factored out the common path to site-packages in a `[constants]` section.